src/main/webapp/app/service/trial.service.ts

```diff
@@ -342,4 +362,20 @@ export class TrialService {
         }
         return result;
     }
+    getErrorResponse(error: HttpErrorResponse, type: string) {
+        if (error.status === 400) {
+            alert('Sorry, your query is invalid.');
```

Comment:
Don't we use `dialogs.notify`?

Reply:
No. I can replace alert to dialogs.notify in next PR.

Reply:
Oh, the dialog is used in oncokb curate not in MM? If it is an easy change, could you add that in?

Reply:
Angular Dialog Service is a complete Angularjs service. I may need to spend time looking for a Angular 2+ service.